### DISCUSSION TITLE
Hurts seeing public commentary like this 😢

### DISCUSSION THREAD

**Discussion by PureKrome** [ravendb/ravendb#14618] [General]
Comment from N.C. (whom I really like, follow and respect) ... which ... is frustrating :(

![image](https://user-images.githubusercontent.com/899878/180808304-aa45687d-436b-4fab-bcb6-7833a83f263a.png)

I really want to know why people think this. Makes me wonder what I'm not getting/understanding/etc...

REF: https://www.youtube.com/watch?v=8IRNC7qZBmk&lc=UgzMqTKAHZjcRTV9lft4AaABAg

Maybe an opportunity for RavenDB to reach out?

**Comment by wallaceturner** [comment]
haters going to hate?? whatever his reasons are there are always going to be different viewpoints/opinions its what makes the world go round.
his grievances may not even be of a technical nature who knows
it would not be worth chasing up imho

**Reply by PureKrome** [reply]
Sure, HG2H. But this is Nick and he's well respected and I _don't feel like_ he just throws troll comments around willy-nilly. 

So maybe this could be a good opportunity to see if would chat to find out his pain points? Maybe he last tried RavenDB back in the 2.x or 3.x days and things are way better now but who knows? or maybe he has some other data that is worth getting feedback on.

I respect him so his opinions take some weight with me. And others, I would assume also.

Or maybe it's as simple as he just doesn't like it for non-technical reasons, which is totally kewl.

**Comment by tobiaszuercher** [comment]
it's unfortunately really sad to see "influencers" doing this. but that's daily life nowadays. crypto influencers seem to influence coding influencers ;)

<img width="832" alt="CleanShot 2022-07-28 at 16 52 29@2x" src="https://user-images.githubusercontent.com/516373/181568261-6b274c54-a6c4-4b2c-83ea-530038a89670.png">

claiming that "it's fine if you don't have any serious load" really doesn't reflect what i've experienced.

it's also very unclear what he means by "data architecture". i can't understand why a db written in .net could be a redflag. ayende actually made talks about why it's actually beneficial to write it in c#. 

apropos "data architecture": i'm  really looking forward to corax & hope it gives even more perf improvements :)

**Reply by ayende** [reply]
To give my two cents on that. 
We have published a whitepaper (https://ravendb.net/whitepapers/couchbase-vs-ravendb-performance-at-rakuten-kobo#accessingTheDataByKey) on a TB range dataset handling 10 K - 30 K queries per second with a latency of under < 200 ms.

We got a Raspberry Pi + SSD drive to hit > 5K requests per second.

I'm not sure what he means by "proper evaluation" or the scenario he was testing. But our numbers and performance testing doesn't seem to indicate any such issues.

There also seem to be certain hostility to .NET based system level software on his part, which I find strange. C# & .NET are actually far better managed platform for system languages than Java for example. And I don't think anyone argues that Java based dbs are not suitable.

**Reply by georgiosd** [reply]
@tobiaszuercher where is this comment? Can you give a link to it please?

**Comment by ayende** [comment]
I'm doing some work (unrelated to this issue), but I thought it would be interesting to show some screen shots:

![image](https://user-images.githubusercontent.com/116915/181825577-723daff3-2d12-479f-aae9-1abecf94f77b.png)

I'm push a dataset that has ~1.3 _billion_ documents on a machine with 8 cores & 64 GB of RAM.
Ingest rate is fair consistent and we are at 125M+ in roughly half an hour.

**Comment by murshex** [comment]
@ayende Dont concern yourself with this nonsense, RavenDB is a fantastic piece of software. Thank you for your hard work

**Comment by georgiosd** [comment]
To me it's strange that someone who is supposed to be a thought leader finds it ok to reference alternatives without naming the reasons or a case study without naming the parameters and architecture.

I have seen benchmarks between technologies where the APIs/capabilities are mapped 1:1 rather than implementing the application in the best manner possible for that technology. For example, if we were comparing EF and RavenDB, denying to do bulk lazy requests because EF can't do it. While getting my PhD I found that this was true of even peer-reviewed journal papers in some instances - a shallow review of anything shows either lack of expertise or a bias towards one of the other options.

Having said all that, we (as the community) are missing tangible case studies that we can reference, rather than depending on the "expert" to make a fair and expert-level evaluation. With the advent of "cloud", the conversation has shifted in the last few years to "scalability" and is even requested where it isn't needed. 

Not going to spill any beans right now, but we (Zero to MVP) plan to create such a public case study with RavenDB in Q4.
